lib/firstFile.js

```diff
@@ -0,0 +1,63 @@
+// Run file in command line - `node lib/firstFile.js`
+
+require('dotenv').config(); // Loads FT API Key from '.env' file in root of project
+var request = require('request'); // Using 'request' package for AJAX requests - https://github.com/request/request
+
+var url = 'http://api.ft.com/content/search/v1';
+var apiKey = process.env.FT_API_KEY; // Get API key
+
+var ApiRequest = function(searchTerm) {
+  this.searchTerm = searchTerm;
+  this.headlines = new Array();
+  // Custom parameters for using search feature of FT API
+  this.params = {
+    "queryString": this.searchTerm,
+    "queryContext": {
+      "curations": ["ARTICLES"]
+    },
+    "resultContext" : {
+      "aspects" : [ "title","lifecycle","location","summary","editorial" ],
+      "maxResults" : "2",
+      "offset" : "0"
+    }
+  }
+  this.options = {
+    method: 'post',
+    body: this.params,
+    json: true,
+    url: url + '?apiKey=' + apiKey
+  }
+
+  this.makeRequest = function() {
+
+    // Passing options property and callback function into `request` function from npm package
+    request(this.options, callback);
+
+    // Tried passing `returnBody` as a callback - not
+    // - returning a TypeError: "returnBody is not a function"
+    function callback(error, response, body, returnBody) {
```

Comment:
if you look at the docs for the `request` library you'll see that the "callback" function only takes 3 arguments 

https://github.com/request/request#super-simple-to-use

```
function (error, response, body) {
}
```

so you've added a 4th argument which isnt given by the request module so it is "undefined"